test/types/records/generic/genericFieldDecl-used.bad

```diff
@@ -1,3 +1,3 @@
 genericFieldDecl-used.chpl:8: In function 'main':
 genericFieldDecl-used.chpl:9: error: this default-initialization is not yet supported
-genericFieldDecl-used.chpl:2: note: field 'x' is declared with a generic type and also a default value
+genericFieldDecl-used.chpl:2: note: field 'x' is declared with a generic type, compiler-generated init, and a default value
```

Comment:
The "compiler-generated init" in here is the compiler generated initializer method, right?  Its placement in the list is a little odd to me - I could imagine someone less familiar with classes/records not knowing the difference between it and "a default value".  I might move that information so that is more separate from the field itself and talks about the enclosing type

Reply:
I just opted to leave out the compiler-generated init distinction.